src/classes/MAPR_RELAutoCreateSettings.cls

```diff
@@ -0,0 +1,107 @@
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group 
+* @group-content ../../ApexDocContent/
+* @description Service class for REL_Relationships_Con_TDTM; used to generate MODL_ConRelationshipAutoCreateMapping
+* models representing Relationship Auto Create mappings set for the Contact object. 
+*/
+public virtual with sharing class MAPR_RELAutoCreateSettings {
+    /***************************************************************************************************************************
+    * @description Instance for Singleton Pattern.
+    ***************************************************************************************************************************/
+    @TestVisible
+    private static MAPR_RELAutoCreateSettings instance;
+
+    /***************************************************************************************************************************
+    * @description Empty constructor is not public for Singleton Pattern.
+    ***************************************************************************************************************************/
+    protected MAPR_RELAutoCreateSettings() {}
+
+    /***************************************************************************************************************************
+    * @description Static method to get the current instance for Singleton pattern.
+    * @return The instance of MAPR_RELAutoCreateSettings.
+    ***************************************************************************************************************************/
+    public static MAPR_RELAutoCreateSettings getInstance() {
+        if (MAPR_RELAutoCreateSettings.instance == null) {
+            MAPR_RELAutoCreateSettings.instance = new MAPR_RELAutoCreateSettings();
+        }
+
+        return MAPR_RELAutoCreateSettings.instance;
+    }
+
+    /***************************************************************************************************************************
+    * @description Retrieves a list of MODL_ConRelationshipAutoCreateMapping representing the Relationship Auto Create
+    * mappings set for the Contact object.
+    * @return A list of ContactRelationshipAutoCreateMapping models.
+    **************************************************************************************************************************/
+    public virtual List<MODL_ConRelationshipAutoCreateMapping> getContactMappingModelsFromSettings(){
+        Map<Schema.SObjectType, List<Relationship_Auto_Create__c>> relAutoCreateListBySObjectType = UTIL_CustomSettingsFacade.getAutoCreateSettingsForContact();
```

Comment:
@MatthewBlanski was this https://github.com/SalesforceFoundation/EDA/pull/1335#discussion_r500080898 done?